examples/FluentUI.Demo.Shared/Pages/Lab/Highlighter/Highlighter.razor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.Fast.Components.FluentUI;
+
```

Comment:
Add // Remember to replace the namespace below with your own project's namespace..

Reply:
Done